invoicing/management/commands/create_sap_xml.py

```diff
@@ -28,7 +28,7 @@ def handle(self, *args, **options):
                 reference_number__in=reference_numbers
             )
         else:
-            installments = installments_with_due_date.sap_pending()
+            installments = installments_with_due_date.sap_ready()
```

Comment:
I can't find `sap_ready()` anywhere, should it be `sending_to_sap_needed()`?

Reply:
Yes it certainly should, good catch 🙂. Fixed.